src/renderers/dom/fiber/ReactDOMFiberComponent.js

```diff
@@ -459,14 +460,16 @@ var ReactDOMFiberComponent = {
 
   mountComponent: function(
     workInProgress : Fiber,
+    tag : string,
     rootContainerElement : Element,
     props : Object,
     context : Object
   ) : Element {
+    // TODO:
     // validateDangerousTag(tag);
-    // workInProgress._tag = tag.toLowerCase();
+    // tag.toLowerCase(); Do we need to apply lower case only on non-custom elements?
```

Comment:
maybe, but we never have special-cased logic for particular custom elements so the code was correct as written – can you add the lowercasing back?